test/verify/check-machines

```diff
@@ -69,200 +67,7 @@ def release_target(used_targets, target):
 @skipImage("Atomic cannot run virtual machines", "fedora-coreos")
 @nondestructive
 @no_retry_when_changed
```

Comment:
I think the `@decorators` on the parent class don't work. The individual test classes repeat them, so that's mostly harmless.